include/seastar/websocket/common.hh

```diff
@@ -92,10 +92,18 @@ protected:
     public:
         connection_sink_impl(queue<buff_t>* data) : data(data) {}
 
+#if SEASTAR_API_LEVEL >= 9
+        future<> put(std::span<temporary_buffer<char>> d) override {
+            return data_sink_impl::fallback_put(d, [this] (temporary_buffer<char>&& buf) {
+                return data->push_eventually(std::move(buf));
```

Comment:
Note: now I think it's bad design to use an extra queue here. There should be one queue, tcp. But not the fault of this series.